Scripts/Data/Operation/TransformDirectionExtractor.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace Zinnia.Data.Operation
+{
+    using UnityEngine;
+
+    /// <summary>
+    /// Extracts the chosen direction.
```

Comment:
```suggestion
    /// Extracts a chosen axis of a <see cref="Transform"/>.
```

Continue using `Extracts the` instead, as we're not unifying the use of `a` vs. `the` so far. (My opinion is to only use `the` when the context is known/explained already. Looking at a type has no other context to it, therefore `a` is more appropriate.)